statsmodels/genmod/generalized_linear_model.py

```diff
@@ -80,8 +84,12 @@ class GLM(base.LikelihoodModel):
         See Parameters.
     family : family class instance
         A pointer to the distribution family of the model.
+    freq_weights : array
+        See Parameters.
     mu : array
         The estimated mean response of the transformed variable.
+    n : array
```

Comment:
`n` is too short as a name. We need a more explicit name.

(not part of documentation changes)